src/components/timeSabarCounter/saberTime.js

```diff
@@ -0,0 +1,77 @@
+import createHTML from './saberTimeStruct';
+
+class Time {
+  constructor() {
+    this.limitTime = null;
+    this.currentTime = 0;
+    this.remainingTime = null;
+    this.clock = null;
+  }
+
+  displayTime(string) {
+    const $timeLeft = document.querySelector('.time');
```

Comment:
I use this notation when a variable holds a reference to DOM element. I rearned it once, but i don't knot if it's necessary.

Reply:
I see. In fact, come to think of it, I have used it myself. I've used it to mark variables that had to do with `jQuery`. Like in the case of `$this = $(this)` to mark `this` accessible to jQuery methods so the names created by you might be associated with `jQuery`. It might be a little confusing but not necessarily.